ci/docker/main-bosh-docker/install.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+set -eu -o pipefail
+
+apt-get update
+
+apt-get install -y --no-install-recommends \
+  apt-transport-https \
+  ca-certificates \
+  curl \
+  git \
+  openssh-client \
+  software-properties-common
+
+curl -o /tmp/download https://storage.googleapis.com/golang/go1.8.3.linux-amd64.tar.gz
+echo "1862f4c3d3907e59b04a757cfda0ea7aa9ef39274af99a784f5be843c80c6772 /tmp/download" | sha256sum -c
+
+tar -xzf /tmp/download -C /usr/local --strip-components=1
+rm /tmp/download
+
+apt-add-repository ppa:brightbox/ruby-ng
```

Comment:
ruby apt only had 1.9.3 and i thought it'd be better to use the same as director